pkg/controller/account/iam.go

```diff
@@ -265,14 +265,16 @@ func CreateIAMUser(reqLogger logr.Logger, client awsclient.Client, userName stri
 // AttachAdminUserPolicy attaches the AdministratorAccess policy to a target user
 // Takes a logger, an AWS client for the target account, and the target IAM user's username
 func AttachAdminUserPolicy(client awsclient.Client, iamUser *iam.User) (*iam.AttachUserPolicyOutput, error) {
+	// grab the ARN base to support govcloud without passing in another argument
+	awsARNBase := strings.Split(*iamUser.Arn, "::")[0]
```

Comment:
Could be handled like the others with `if config.IsFedramp() ` instead of a string split?